client/core/client.go

```diff
@@ -142,36 +139,33 @@ func DefaultDiscoveryConfiguration() DiscoveryConfiguration {
 		MulticastAddressUDP4: DiscoveryAddressUDP4,
 		MulticastAddressUDP6: DiscoveryAddressUDP6,
 		MulticastOptions: []coapNet.MulticastOption{coapNet.WithMulticastInterfaceError(func(iface *net.Interface, err error) {
-			name := ""
-			if iface != nil {
-				name = iface.Name
-			}
-			log.Debugf("network interface '%v': %w", name, err)
+			// ignore error
```

Comment:
too many logs or why not at least keep `logger.Debug` here?

Reply:
We don't have `logger` there so by default it is disabled -> if someone wants to debug, he can change it.